### ISSUE TITLE
Component tokens for v1

### ISSUE THREAD

**Issue by mimarz** [digdir/designsystemet#2488]
_This is a summary of meeting held on Component tokens for v1._

## TLDR

We decided to keep todays approach of only having component tokens in CSS for v1. 

The door is open for component tokens in Figma in the future if there is a big enough want from the community. At the moment we don't want to introduce more complexity and maintenance to our existing design-tokens pipeline as users can still accomplish desired results with manually editing Figma and CSS (such as border-radius).

We will document some examples of how users should use component tokens in CSS.

### Notes
- Todays solution has component tokens in CSS as CSS variables (semantic tokens).
- If we want component tokens in Figma, it would also impact design-tokens pipeline.
- Todays solution is not a blocker for users wanting to tailor aspects of specific components. It can be achieved manually by editing the component in Figma and replicating the change in custom CSS (or CSS component token).
- `border-radius` adjustment was used as an example for component token.
- Can be added post-launch as its mostly an addition of more tokens.
- Multiple color-schemes will be solved independently from this with existing design-tokens pipeline. 

### Pro
- Component tokens in Figma (and design tokens) would open a new level of granularity.
- This could be beneficial for making it easier to tailor available aspects of specific components and have it synced to code with our design-tokens pipeline.
- Component tokens could be limited to only a certain amount of components.

### Cons
- More design-tokens with proportionate amount of maintenance needed in design-tokens pipeline.
- No overwhelming need from community (yet?)
- Component tokens would scale proportional. Example, 33 components * 3 component tokens = 99 additional tokens.
- Figma/Token Studio has had performance issues with to many variables/tokens.
- If we open the door for component tokens, it can quickly balloon into feedback for more component specific tokens, such as spacing, gap, sizes?
- Previous experience with component tokens getting out of hand with small hacks being added here and there.

#### Legend:
- Design-tokens pipeline; All our tools and products related to design-tokens;
  - Figma community file
  - Designsystemet figma plugin
  - Token build script in cli
  - Token studio to Figma variables support (in json)
  - Design-tokens (in json)
